defaults/main.yml

```diff
@@ -11,33 +11,24 @@ users:
-  - name: tug76662
-    authorized:
-      - https://github.com/relaxing.keys
+      - https://github.com/bibliotechy.keys
   - name: tuf73699
     authorized:
-      - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCpPIvJcG07yr/CVPk/dYW5geeGvvzZ3yG91Z3/QGtGeuovwb1dtklWMAXAqrx9gT8B80Q/HF4uuWTi+lf6ja46ig7JsLsHXg9esnFmFjtnqlqUP4jFu+i2klBsE/fdFG9FVCYeXxrJMhR4qVhFj4o+jldeALvHuEw7AINSvVlMpSaZ14YeY2goKy52LW9xcVrY2idVMTbAftU7pOmmFDvjMsURnUdYOaQhrAWH8fCiIapi2lMO7dGl0TPj5r5yR5Cjw/gXcgL/sU6nPmTNh50Gni4NYUrcF9CkLRCjfLU5T7dMYcrL/mXtek8s/rVMNTw/Du1ZWgWhjYVkuIJvc80n tuf73699@lib-pal-dli-sn.lib.temple.edu"
-      - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDmlgPlDQ1CNJ/kCPc6sqpDalE1xnifCn++4GVKKl+EBiScmnWuVMQ9zUFOqhA667f6j3NEFzAbRwnNEVvbg7BqX56COwCvxrKR/fwkexKRtvHVHAGLFtZ8Ng3aIimzAw3nURWIGuUTciH2m1N/Npo1UuSY/VvDuhhg7mNgGqIs6OvH55t7No3eyq/DceVJCtPacd6ygwCfGyFAEmxfwT1Xh1udZKx21MdlBWMkCwuV/fnwm6uLD8WUBeXtUIIBlpNUcFkTIL1cSUkjKPqRQ67OFR2GJ+4k2EuT/E5wwfh5ltnIgd2N3CrvV40P5RvPzG0KMkiSwN7kffcwneM3efmv skng@80-e6-50-18-81-da.wireless.temple.edu"
-  - name: tud19265
-    authorized:
-      - "ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAgEAo4TneEhdTuIHcHbFv6ACK12440Hr/fqntq2D2/+38lLGxHs7NcQi1EplNSxQYzTb7BkcUeI+b9ifWrp58nj6SlNexETfRLrs0wOL7EwFJb6loWkyKEs6NkztcufAjcC8W08q3onfEUoBH7kCq3De5vk5CzlsOif4Rj0tZyB/xYqO3f96bbQ0w6c8Ki3Me5LYF3bMuoyi+z0Tuxqoa8DkZWjafs5XdPF1fcZObHErAZL8mLzRG2MUhijszSkLkVazNffjCY/RygcPIRQqrFS/FJRNItXUYkPkPY+JIQhPJ5WjWnVro6bmD1p+oIWgqAArh7CKvEeQU7EXHs7+0/FbLoHVygeLOTjzqFQSetFDJdHv7pF0TuxNVJU2hJkneKv1L6aGvRQ+NnAJjSt31M2CD1OeecEwmyiI33wYP4eCg3PI2XVnRWQhvC3z5+1rFmrISFByZI5zsuiQYvHWPKuuZrhICUy23SDsmASxgXOG6BDNWUjTS0wvvMhuba1LjzujmbObA2aPRXO6C1JqKEgFJZmfMqPIL0U5L4lQcivMgzGr6nNGOqOhfB//9fZYLCJi/jKFDkJxwsPkUO+9DIcA7ehqrSHgtLAC5gmYlHJYDWrKQjmlJr7UT5LZdHQ23L4/hBHjsy2mmdN3g4XTIwjowHJMI0SVdy8n/naJs0ZFqvs= rsa-key-20161215"
-  - name: tul08567
-    authorized:
-      - https://github.com/cmharlow.keys
+      - https://github.com/nomadicoder.keys
   - name: tuh42082
     authorized:
       - https://github.com/sensei100.keys
   - name: conan_the_deployer
     authorized:
       - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDNdyy/Nv2sbf2jwhTksqQSHvCAQ9+8BrZApCAu6N80oMqPGC9oeohrNcr2zonL+K5LsWxxtxNIQDB2qlo5JR/jP/vBTyzUmxU1B7FP50a2JyziLxzAQUPdVp9zdnv/bncyhJCHo1+XWfy6ZWpNV2ejumHjeAqiIkNGg5jB0JuOo0umiyIS+v1mh2G0kwSsf1rIz//avlLqhydQQfOUR6fp635N2kZYgPMOE/07beNAxQHy6zRq/tCvlBmUhJg8Sm/STnpevKMUV9xcI1IFLNtbHFUPdvGjdFKplRL8Bn28Zzja92aYZ7miNKQiNL1sFyxmwSkKEOdlnvZzfX6quCIH6bEGz2vZPJRGhPav455Iu6/HbWzjOlqF0sK9pOYKHBFSk6sSWrgAT6eg325nXjLzNIQLvlbq5EQsxP3Tr5bkVPPd05PmTbtrYr8jv7HmPp59D2o8qJWXK1mSegegvYtXcowFZXD5o6AyhoAb4IMPNWOMgWMreWK8WPbkFLf6L4WYV3i92v92HcfVddhWy74G2V7zIK/77IR7lYZZSlFmv6UD/fhnuw6bouPpE08TH5aTOT5wg9V3MkfgviJoALECjwdLSo6KFRkeUimKWIe7KvRyY6UHYzs5PZJEpShyuvnkJm9t2ejseStLP+dRHcLiCyBRta2RCcLTx7JkXC416Q== conan_the_deployer@temple.edu"
```

Comment:
@bibliotechy Should mine also be changed to use keys?

Reply:
I think yours already was, so it didn't require a change